include/irr/asset/format/EFormat.h

```diff
@@ -1007,12 +1014,27 @@ namespace asset
                 blockByteSize(getTexelOrBlockBytesize(format))
             {}
             
+            //! It converts input texels strides to compute a true extent of an image
+            /*
+                The true extent is a dimension of stride in texels or in blocks, depending
+                of what are you dealing with.
+                @param coord it's a dimension in texels.
+            */
 
 			inline auto convertTexelsToBlocks(const core::vector3du32_SIMD& coord) const
 			{
 				return (coord+maxCoord)/dimension;
 			}
 
+            //! It converts input texels strides to compute length-block size
```

Comment:
nope, rounds up measurements in `texels` to mutliples of block sizes

Reply:
adjusted in following commit